driver/normalizer/normalizer.go

```diff
@@ -211,7 +210,7 @@ var Normalizers = []Mapping{
 							"Name":      Var("local"),
 							"Node": UASTType(uast.Identifier{}, Obj{
 								uast.KeyPos: Any(),
-								"Name": String("."),
+								"Name": Any(),
```

Comment:
FYI it just drops the name of the imported type from an AST.

Reply:
thank you for the heads up! Yup, if left this way for now (which is anyway better then before) it needs to have at least a `TODO`

Reply:
added in 7b65c80